frontend/packages/operator-lifecycle-manager/src/components/operator-install-page.tsx

```diff
@@ -0,0 +1,484 @@
+  referenceForGroupVersionKind,
+} from '@console/internal/module/k8s';
+import {
+  ClusterServiceVersionModel,
+  InstallPlanModel,
+  PackageManifestModel,
+  SubscriptionModel,
+} from '../models';
+import {
+  CRDDescription,
+  ClusterServiceVersionKind,
+  SubscriptionKind,
+  InstallPlanKind,
+  PackageManifestKind,
+} from '../types';
+import {
+  GreenCheckCircleIcon,
+  RedExclamationCircleIcon,
+  YellowExclamationTriangleIcon,
+} from '@console/shared/src/components/status/icons';
+import { InstallPlanPreview } from './install-plan';
+import { RedExclamationCircleBadge } from '@console/shared/src/components/badges/RedExclamationCircleBadge';
+
+const getCRDFromObject = (obj: ClusterServiceVersionKind | InstallPlanKind) => {
+  return obj?.metadata?.annotations?.['operatorframework.io/initialization-resource'];
+};
+
+const isCRDAvailable = (obj: ClusterServiceVersionKind | InstallPlanKind) =>
+  typeof getCRDFromObject(obj) !== 'undefined';
+
+const installPageResources = (namespace: string) => [
+  {
+    kind: referenceForModel(ClusterServiceVersionModel),
+    namespaced: true,
+    fieldSelector: `metadata.name=${new URLSearchParams(window.location.search).get('csvName')}`,
+    isList: true,
+    namespace,
+    prop: 'clusterServiceVersions',
+  },
+  {
+    kind: referenceForModel(ClusterServiceVersionModel),
+    fieldSelector: `metadata.name=${new URLSearchParams(window.location.search).get('csvName')}`,
+    isList: true,
+    namespace: 'openshift-operators',
+    prop: 'allClusterServiceVersions',
+  },
+  {
+    kind: referenceForModel(SubscriptionModel),
+    namespaced: false,
+    fieldSelector: `metadata.name=${new URLSearchParams(window.location.search).get('pkg')}`,
+    isList: true,
+    optional: true,
+    prop: 'subscription',
+  },
+  {
+    kind: referenceForModel(InstallPlanModel),
+    prop: 'installPlan',
+    namespaced: false,
+    isList: true,
+    optional: true,
+  },
+  {
+    isList: true,
+    kind: referenceForModel(PackageManifestModel),
+    namespace: new URLSearchParams(window.location.search).get('catalogNamespace'),
+    fieldSelector: `metadata.name=${new URLSearchParams(window.location.search).get('pkg')}`,
+    selector: {
+      matchLabels: {
+        catalog: new URLSearchParams(window.location.search).get('catalog'),
+      },
+    },
+    prop: 'packageManifest',
+  },
+];
+
+const ViewInstalledOperatorsButton: React.FC<InstallPageMessageButtonProps> = ({
+  namespace,
+  className,
+}) => (
+  <Button
+    className={className}
+    data-test-id="view-installed-operators-btn"
+    variant="link"
+    component="a"
+    href={`/k8s/ns/${namespace}/${referenceForModel(ClusterServiceVersionModel)}`}
+  >
+    View Installed Operators
+  </Button>
+);
+
+const InstallFailedMessage: React.FC<InstallPageMessageProps> = ({ namespace, csvName, obj }) => {
+  const crdMessage = isCRDAvailable(obj)
+    ? 'The required custom resource will need to be created in the operator once resolved.'
```

Comment:
I'm not sure what this message means. @itsptk What is this message trying to say?

Reply:
Let's leave this as is and we can tweak it later if it is still confusing.

Reply:
@itsptk  - Peter, can you suggest a tweak to the wording for when an error occurs and the user will need to still make a CR.

Reply:
Agreed, rereading this it isn't very clear. It's attempting to say 'this process would have had you create the required CR, but now that the install failed you need to first fix the operator install then create the CR on your own from the operator details.'  Is there any way the user can fix the installation aside from just reinstalling the operator? 

If not, then the message probably doesn't even need to mention instantiating the CR since the whole process is going to be restarted anyway.

If so, than maybe:
"The operator did not install successfully. The required custom resource can be created in the Operator's details view."

cc @tlwu2013

Reply:
Made the change.